src/js/components/DataTable/Body.js

```diff
@@ -17,6 +18,7 @@ const Body = forwardRef(
       border,
       columns,
       data,
+      gdtSelected,
```

Comment:
"gdt"?

Reply:
"grommet data table". I chose a name that wouldn't have a potential collision with names that the user may select for new columns.